packages/lesswrong/server/fmCrosspost/denormalizedFields.ts

```diff
@@ -28,7 +32,7 @@ export const isValidDenormalizedData = (payload: unknown): payload is Denormaliz
   hasStringParam(payload, "title") &&
   hasBooleanParam(payload, "isEvent") &&
   hasBooleanParam(payload, "question") &&
-  hasStringParam(payload, "url");
+  hasOptionalStringParam(payload, "url");
```

Comment:
probably delete `isValidDenormalizedData` entirely, since it's not actually used anywhere